<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pad_configure</class>
 <widget class="QDialog" name="pad_configure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>720</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="Devices" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>231</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QComboBox" name="Interesting_Devices">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Devices</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Triangle_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>160</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="Triangle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Triangle</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Dpad_Up_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="Dpad_Up">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dpad Up</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Dpad_Left_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="Dpad_Left">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dpad Left</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Dpad_Right_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="Dpad_Right">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dpad Right</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Dpad_Down_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="Dpad_Down">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dpad Down</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Square_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>260</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="Square">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Square</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Circle_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>260</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="Circle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Cross_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>360</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="Cross">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cross</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Select_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>101</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="Select">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>52</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Start_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>609</x>
     <y>250</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <widget class="QPushButton" name="Start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>pad_configure</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>pad_configure</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
